tests/conftest.py

```diff
@@ -89,7 +89,7 @@ def testbed_devices(ansible_adhoc, testbed):
         # when no ptf defined in testbed.csv
         # try to parse it from inventory
         dut = devices["dut"]
-        ptf_host = dut.host.options["inventory_manager"].get_vars(dut.hostname)["ptf_host"]
+        ptf_host = dut.host.options["inventory_manager"].get_host(dut.hostname).get_vars()["ptf_host"]
```

Comment:
According to PR description it looks like an bug in new ansible.
Could you put a comment saying that this is changed according to W/A descirbed in https://github.com/ansible/ansible/issues/33684?

Reply:
This is not a W/A. `get_vars` was removed from the inventory manager.

https://github.com/ansible/ansible/commit/cea7151a5ba1a5339e828534ddd6000347867dea#diff-eaa1d70fa6781d189e1c19b20913f517